src/prommis/nanofiltration/diafiltration_two_salt.py

```diff
@@ -259,6 +264,27 @@ class TwoSaltDiafiltrationData(UnitModelBlockData):
 """,
         ),
     )
+    CONFIG.declare(
+        "membrane_width",
+        ConfigValue(
+            default=1,
+            doc="Width of the membrane (x-direction) in meters",
+        ),
+    )
+    CONFIG.declare(
+        "membrane_length",
+        ConfigValue(
+            default=100,
+            doc="Length of the membrane, wound radially in meters",
+        ),
+    )
+    CONFIG.declare(
+        "applied_pressure",
+        ConfigValue(
+            default=10,
+            doc="Pressure applied to membrane in bar",
+        ),
+    )
```

Comment:
These don't need to be config entries. You can just initialize the `Vars` at those values.